packages/core/src/drizzle/index.ts

```diff
@@ -225,7 +219,8 @@ export const onchainTable = <
   extra: extra;
   dialect: "pg";
 }> => {
-  if (instanceId === undefined) {
+  // @ts-ignore
+  if (globalThis.INSTANCE_ID === undefined) {
```

Comment:
It should be safe to do `const instanceId = globalThis.INSTANCE_ID` in the file scope, right?